controllers/home-routes.js

```diff
@@ -70,6 +70,13 @@ res.render('login')
 
 })
 
+//Direct to signup screen
+router.get('/signup', (req, res) => {
```

Comment:
Should probably add a feature to not be able to sign-up if already logged in, but non-issue for this merge IMO.